frontend/content/charts/outputs-opreturn-bytes.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "OP_RETURN Output Bytes"
+draft: false
+author: "deadmanoz"
+categories: "Outputs"
+position: 6
+tags: [op_return]
+thumbnail: outputs-opreturn-bytes.png
+chartJS: outputs-opreturn-bytes.js
+images:
+  - /img/chart-thumbnails/outputs-opreturn-bytes.png
+---
+
+Shows the total data bytes in OP_RETURN outputs per day.
+<!--more-->
+
+OP_RETURN outputs allow for the inclusion of arbitrary data in an OP_RETURN transaction output (following the OP_RETURN op code).
+By definition, OP_RETURN outputs are provably unspendable so can safely be pruned by a Bitcoin node.
+This chart tracks the total volume of data stored via OP_RETURN outputs daily.
```

Comment:
“Pruned” is a loaded term that has caused a lot of confusion in the past few months in the context of “OP_RETURN outputs being prunable [from the UTXO set]”. They’re not unspendable “by definition”, but by consensus. ;)

How about:

```suggestion
Datacarrier outputs (aka OP_RETURN outputs or nulldata outputs) allow for the inclusion of arbitrary data in an output script starting (following the OP_RETURN opcode).
OP_RETURN outputs are provably unspendable, so can be safely dropped from the UTXO set by a Bitcoin node.
This chart tracks the total volume of data bytes stored via OP_RETURN outputs daily.
```